packages/yourloops/pages/hcp/patients/cards.tsx

```diff
@@ -87,8 +87,9 @@ const patientListStyle = makeStyles(theme => ({
 const removeButtonEnabled = false;
 
 function PatientCard(props: PatientElementCardProps): JSX.Element {
```

Comment:
I suggest to completely remove this cards, it doesn't fit with yourloops 3.0 design

Reply:
Agreed, I would prefer to do this in another PR if that's ok with you?